paasta_tools/autoscaling_lib.py

```diff
@@ -344,6 +344,16 @@ def humanize_error(error):
         return 'utilization within thresholds'
 
 
+def is_task_healthy(task):
+    """
+    Check a marathon task has healthcheck results
+    and check that they are all healthy.
+    """
+    if task.health_check_results:
+        return all([hcr.alive for hcr in task.health_check_results])
```

Comment:
Good shout @solarkennedy I'll consolidate that down tomorrow 👍 

Also spotted https://github.com/Yelp/paasta/blob/369273ba8944d88b80517eca3ef2f2f3f80e1339/paasta_tools/check_marathon_services_replication.py#L196 now that I've grep'd the repo. Although that seems to be ignoring young tasks, presumably to prevent firing alerts on brand new tasks that are starting? One minute seems kinda arbitrary though...